.github/workflows/docker.yaml

```diff
@@ -10,7 +10,7 @@ on:
       - main
 
 env:
-  SUI_GIT_REVISION: devnet-0.23.0
+  SUI_GIT_REVISION: a363182 # Testnet Wave 2
```

Comment:
we don't offer devnet docker anymore?

Reply:
This is just gonna tell the CI to build and push a new image. It's not gonna delete any existing ones.